rosidl_generator_py/resource/_msg.py.template

```diff
@@ -0,0 +1,66 @@
+# generated from rosidl_generator_py/resource/_msg.py.template
+# generated code does not contain a copyright notice
+
+class @(spec.base_type.type)__Constants(type):
+@[for constant in spec.constants]@
+
+    @@property
+    def @(constant.name)(cls):
+        """Message constant '@(constant.name)'"""
+        return @constant_value_to_py(constant.type, constant.value)
+@[end for]@
+
+    def __setattr__(cls, name, value):
+        raise AttributeError("Class variables are read-only")
+
+    @@classmethod
+    def __prepare__(cls, name, bases, **kwargs):
+        return {
+@[for constant in spec.constants]@
+            '@(constant.name)': @constant_value_to_py(constant.type, constant.value),
```

Comment:
Is each constant and its value mentioned twice, once in this line and once in line 5-11?